<ui version="4.0" >
 <class>AddBuddyDia</class>
 <widget class="QDialog" name="AddBuddyDia" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>add_buddy</string>
  </property>
  <property name="styleSheet" >
   <string>QDialog {
 
background-color:rgb(246,243,243);

}
</string>
  </property>
  <property name="modal" >
   <bool>false</bool>
  </property>
  <widget class="QRadioButton" name="RB_mobile" >
   <property name="geometry" >
    <rect>
     <x>13</x>
     <y>20</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>mobile no</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="RB_fetionID" >
   <property name="geometry" >
    <rect>
     <x>12</x>
     <y>50</y>
     <width>89</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>fetion id</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ED_mobile" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>20</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ED_fetionID" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>50</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="frame" >
    <bool>true</bool>
   </property>
   <property name="readOnly" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>223</y>
     <width>261</width>
     <height>91</height>
    </rect>
   </property>
   <property name="autoFillBackground" >
    <bool>false</bool>
   </property>
   <property name="title" >
    <string>send invite</string>
   </property>
   <property name="flat" >
    <bool>false</bool>
   </property>
   <property name="checkable" >
    <bool>false</bool>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>i am</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ED_usr_name" >
    <property name="geometry" >
     <rect>
      <x>40</x>
      <y>23</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>140</x>
      <y>30</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>ï¼Œnow using fetion</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>want to make friend with you</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>120</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title" >
    <string>contact info</string>
   </property>
   <widget class="QComboBox" name="CB_group" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>22</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="BT_newgroup" >
    <property name="geometry" >
     <rect>
      <x>175</x>
      <y>22</y>
      <width>80</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text" >
     <string>new group</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>26</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>group</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5" >
    <property name="geometry" >
     <rect>
      <x>12</x>
      <y>67</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ED_localName" >
    <property name="geometry" >
     <rect>
      <x>63</x>
      <y>63</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="BT_getPersInfo" >
   <property name="geometry" >
    <rect>
     <x>18</x>
     <y>80</y>
     <width>80</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>pre info</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox" >
   <property name="geometry" >
    <rect>
     <x>102</x>
     <y>319</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="add_state" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>90</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
